### ISSUE TITLE
Output verbosity

### ISSUE THREAD

**Issue by silverwind** [jedmao/eclint#84]
The current output seems hard to parse visually for me:

<img width="881" src="https://user-images.githubusercontent.com/115237/28751947-7314358a-7514-11e7-83ab-a1e1179bc757.png">

All I care about is filename, line number, column and a short and descriptive error. `eslint` is a good example:

<img width="656" alt="screen shot 2017-07-30 at 10 48 20" src="https://user-images.githubusercontent.com/115237/28751971-a4478616-7514-11e7-97f7-b0c5b3730a92.png">

Maybe it'd be good to have the minimal output without an argument and introduce `--verbose` for the current output.

**Comment by gucong3000** [comment]
LGM

**Comment by gucong3000** [comment]
Upstream: gucong3000/gulp-reporter#105

**Comment by silverwind** [comment]
Thanks! That looks like a good first step. I'd also remove the URL to the rule, so 

```
    [03:01] ❌️ invalid indent size: 5, expected: 9 (EditorConfig indent_size https://github.com/editorconfig/editorconfig/wiki/EditorConfig-Properties#indent_size)
```
could become
```
    [03:01] ❌️ invalid indent size: 5, expected: 9    indent_size
```
With `indent-size` in a different color, maybe similar to how `eslint` does it.

**Comment by gucong3000** [comment]
Fixed in #92

**Comment by silverwind** [comment]
Great work! I really like the now concise output. 👍
